README.md

```diff
@@ -81,17 +81,17 @@ To unleash the power of Pythoshop, follow the simple steps below:
 ```python
 from pythoshop import adjust_brightness, resize_image, transform_image, apply_filter
```

Comment:
should we include the import for mpimg: matplotlib ?

Reply:
also the badges, CI-CD one and test coverage, repostatus badge, will be added in this PR or a different one?

Reply:
I was planning on having the CI-CD one/test coverage, repostatus badge updates on a different PR since they're unrelated to the change of input parameters from image_path to image. Matplotlib is necessary for input though and does need to be changed , thank you!!!